auth_cas/tests/test_auth_cas.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016-2018 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests import common
+from mock import patch
+from ..controllers.main import Controller
+
+
+class TestAuthCas(common.TransactionCase):
+
+    def setUp(self):
+        """Create a packaging and wac."""
```

Comment:
looks a strange docstring for this project

Reply:
indeed, please adapt to something simple from Odoo standard